radixdlt/src/main/java/com/radixdlt/mempool/messages/MempoolAtomMessage.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * (C) Copyright 2020 Radix DLT Ltd
+ *
+ * Radix DLT Ltd licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except in
+ * compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied.  See the License for the specific
+ * language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.mempool.messages;
+
+import java.util.Objects;
+
+import org.radix.network.messaging.Message;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.radixdlt.common.Atom;
+import com.radixdlt.serialization.DsonOutput;
+import com.radixdlt.serialization.DsonOutput.Output;
+import com.radixdlt.serialization.SerializerId2;
+
+@SerializerId2("message.mempool.atom")
+public class MempoolAtomMessage extends Message {
```

Comment:
Might be more self-explanatory as "MempoolAtomAddedMessage", though that is quite the mouthful I must admit..

Reply:
agreed, `MempoolAtomAddedMessage` is a better name :)

Reply:
OK, will change.